dist/cjs/src/okx.js

```diff
@@ -3863,7 +3863,7 @@ class okx extends okx$1 {
             'amt': this.numberToString(amount),
         };
         const networks = this.safeValue(this.options, 'networks', {});
-        let network = this.safeStringUpper(params, 'network'); // this line allows the user to specify either ERC20 or ETH
+        let network = this.safeString(params, 'network'); // this line allows the user to specify either ERC20 or ETH
```

Comment:
@dmatora You don't need to commit this file, but don't worry I will revert it this time

Reply:
@carlosmiei thanks